test/integration/model/nested_multiple_elements_error_world/model.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
```

Comment:
I don't think you need this file

Reply:
Removed and moved the sdf file to the sdf directory under `nested_multilpe_elements_error_world.sdf`